draft-ietf-tls-tls13.md

```diff
@@ -1165,6 +1165,13 @@ fatal "record_overflow" alert.  This limit is derived from the maximum
 TLSPlaintext length of 2^14 octets + 1 octet for ContentType + the
 maximum AEAD expansion of 255 octets.
 
+There is a limitation on the total number of bytes that can be transmitted
+under one set of keys. For the AES-GCM ciphersuites this limit is 
+2^36 bytes. For ChaCha20-Poly1305 this limit is larger then the size of the
```

Comment:
Just for clarity, can we add "(64 GiB)" after "2^36 bytes"? (maybe also below) I know it's redundant, but this is a relatively small value from some perspectives, so I think it's worth stressing.